docker/Dockerfile.nightly-cpu

```diff
@@ -3,20 +3,22 @@ FROM ubuntu:18.04
 ENV PYTHONUNBUFFERED TRUE
 
 RUN apt-get update && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    build-essential \
-    libatlas-base-dev \
-    libopencv-dev \
-    graphviz \
-    python-dev \
+    DEBIAN_FRONTEND=noninteractive apt-get install --no-install-recommends -y \
+    fakeroot \
+    ca-certificates \
+    dpkg-dev \
+    g++ \
     python3-dev \
     openjdk-8-jdk-headless \
     curl \
     vim \
     && rm -rf /var/lib/apt/lists/* \
     && cd /tmp \
     && curl -O https://bootstrap.pypa.io/get-pip.py \
-    && python get-pip.py
+    && python3 get-pip.py
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+RUN update-alternatives --install /usr/local/bin/pip pip /usr/local/bin/pip3 1
 
 RUN pip install --no-cache-dir --pre mxnet-model-server \
```

Comment:
dont we need pip3 to install these packages?

Reply:
I created alternative for pip -> pip3